app/views/admin/items/new.html.erb

```diff
@@ -9,10 +9,20 @@
 
   <%= f.label :price%>
   <%= f.text_field :price%>
+# I've tried this a few times with different f. combos. Either nothing renders or there are a variety of nil class/merge/no conversion to key errors
 
+<%= form_for(@item) do |f| %>
   <%= f.label :category %>
-  <%= select_tag :category_id, options_from_collection_for_select(@categories, :id, :name) %>
-  <%= f.collection_select :category_id, @categories, :id, :name, include_blank: true %>
-  <%= f.collection_select :item, :category_id, @categories, :id, :name, include_blank: true %>
+  <select class="browser-default" name= "item[category]">
```

Comment:
I've had so many issues trying to get drop-downs to work with the rails helpers.